values/src/main/java/org/spine3/net/Schemas.java

```diff
@@ -25,12 +25,23 @@
 
 import java.util.Map;
 
+/**
+ * Utility class for working with {@link Schema}.
+ *
+ * @author Mikhail Mikhaylov
+ */
 @SuppressWarnings("UtilityClass")
 /* package */ class Schemas {
 
     private Schemas() {
     }
 
+    /**
+     * Parses schema from String representation.
+     *
+     * @param value String schema representation.
+     * @return {@link Schema} value
+     */
     /* package */ static Schema of(String value) {
```

Comment:
As the documentation says, it “parses” the value. So, the method name change would be appropriate.